docs/usage/manual_cleanup.md

```diff
@@ -0,0 +1,105 @@
+# Manual cleanup of Shoot cluster and associated kubernetes and infra resources
+
+Though gardener takes care of graceful deletion of all hosted resources, even for non-healthy or cluster with creation/reconciliation error; sometimes, may be because of external action or some unknown issue, specifically in case something goes wrong with the local setup for development, we have to delete the cluster and cleanup all the resource manually, to save cost and not have issue while creating new shoot mostly with same name on same landscape. This document provides the list of resources to be cleaned up manually to remove the complete trace of the shoot cluster.
+
+## Terminology
+
+- __ShootTechnicalID__ : The value of `shoot.status.technicalID` in shoot resource YAML, typically equals to _"shoot--[project-name]--[shoot-name]"_
+- __ShootUid__ : The value of `shoot.status.uid` in shoot resource YAML.
+
+## Kubernetes resource
+As a part of shoot creation, gardener creates some kubernetes resource on Seed and virtual-garden cluster. We have to clean this up so that gardener doesn't waste cpu cycles in reconciling them.
+
+### On virtual-garden cluster:
+- Delete the `shoot.core.gardener.cloud` resource from project namespace.
+- Delete the `backupentry.core.gardener.cloud` resource from project namespace.
+
+### On seed cluster:
+- Delete the namespace with name equivalent to `ShootTechnicalID` from seed cluster on which shoot was hosted.
+    - Remove the finalizers on the namespace scoped resources from that namespace to let namespace deletion succeed and not stuck in terminating state.
+- Delete the associated cluster scoped resource `backupentry.extension.gardener.cloud` with name `ShootTechnicalID--ShootUid` hosted on the referred seed along with the referred `secret` resources.
+
+## Cloud provider resource
+
+As a part of Shoot hosting, gardener create different cloud providers resources like vpc, vm, object in objet store, etc. Missing out on cleaning up some of these resource may end up costing you unnecessary. These resources are spread up across max three accounts one for shoot cloud provide account, dns account, and object store account though usually all might be same for development purpose. Please follow the doc to cleanup all cloud provider resources.
+
+### Shoot cloud provider account
+
+Gardener as a part of shoot reconciliation create the different resource in end-users account. i.e the shoot owners account.
+
+#### AWS
+- Delete the VPC with name `ShootTechnicalID`.
```

Comment:
VPC is optional and it depends whether shoot was deployed into preexisting VPC or Gardener has created it. Please, note in the documentation.